demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -71,7 +71,9 @@ def _content_files_list_generator_factory(self, dir_name: str, suffix: str) -> I
             content_object = path_to_pack_object(object_path)
             # skip content items that are not displayed in the id set, if the corresponding flag is used
             if self._filter_items_by_id_set and content_object.type().value not in [FileType.RELEASE_NOTES.value,
-                                                                                    FileType.RELEASE_NOTES_CONFIG.value]:
+                                                                                    FileType.RELEASE_NOTES_CONFIG.value,
+                                                                                    FileType.TEST_PLAYBOOK.value]:
```

Comment:
I suggest adding a comment describing why we exclude both ReleaseNotes and TestPlaybooks at this point (since they are missing from the id set but are needed in the pack's zip), for better readability of this code.